pytripgui/view/plan.ui

```diff
@@ -28,52 +28,36 @@
      <property name="currentIndex">
       <number>0</number>
      </property>
-     <widget class="QWidget" name="tab_5">
+     <widget class="QWidget" name="general_tab">
       <attribute name="title">
        <string>General Info</string>
       </attribute>
-      <layout class="QGridLayout" name="gridLayout_5">
+      <layout class="QFormLayout" name="formLayout">
```

Comment:
![Screenshot_2019-07-12_12-10-30](https://user-images.githubusercontent.com/7140398/61120993-77033e00-a49e-11e9-9886-98ee165b180a.png)
Spanning the spinbox is not a good layout. The "up-down" are very far from the numbers, and generally it makes the interface look more cluttered.

Reply:
maybe quick fix to put in spacers to the right?

Reply:
One more issue: Max min ganry angle is limited to [0.0 ; 99.99] which is not enough.
We need [-400; +400] to cover multiple vendors.

Funny detail:  +370° != +10° since going from +359° to 370° is faster than going from +359° to +10° where the ganry would have to move all the way back. This is called "overtravel".

Reply:
Same for couch.

Reply:
![image](https://user-images.githubusercontent.com/47323191/61177909-a28e4180-a5e0-11e9-930d-fe8ab962126a.png)
How about that?

Reply:
![image](https://user-images.githubusercontent.com/47323191/61177918-1d575c80-a5e1-11e9-8c42-7af3fdad3751.png)
have you seen plan setup as well?